content/docs/user-guide/external-outputs.md

```diff
@@ -1,51 +1,48 @@
-# Managing External Data
+# External Outputs
```

Comment:
Apparently this name was quite intentional as "outputs" is not meaningful for end-users (at the user guide level)...

Reply:
Reverted the name. Let's just try to merge the copy edits then? Thanks!